clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -1905,7 +1905,8 @@ void ConvertCIRToLLVMPass::runOnOperation() {
                CIRToLLVMVecTernaryOpLowering,
                CIRToLLVMComplexCreateOpLowering,
                CIRToLLVMComplexRealOpLowering,
-               CIRToLLVMComplexImagOpLowering
+               CIRToLLVMComplexImagOpLowering,
```

Comment:
I missed this in previous reviews, but we were trying to keep these in lexicographical order.